src/templates/mongoose/templateHelpers.ts

```diff
@@ -0,0 +1,24 @@
+import { mongoosePrimitiveTypes } from '../../model/dataTypes/primitiveDataTypes';
+import DataType from "../../model/dataTypes/DataType";
+
+export function getMongoosePrimitive(type: DataType): string {
+  // TODO: this is a workaround for the typeName holding a normalized name instead of an original name
+  let typeName = type.typeName.replace("Primitive", "").toLowerCase();
```

Comment:
Codacy found an issue: ['typeName' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5738534)